search/search_repository.go

```diff
@@ -346,6 +346,6 @@ func (r *GormSearchRepository) SearchFullText(ctx context.Context, rawSearchStri
 func init() {
 	// While registering URLs do not include protocol becasue it will be removed before scanning starts
 	// Please do not include trailing slashes becasue it will be removed before scanning starts
-	RegisterAsKnownURL("work-item-details", `(?P<domain>demo.almighty.io)(?P<path>/detail/)(?P<id>\d*)`)
-	RegisterAsKnownURL("localhost-work-item-details", `(?P<domain>localhost)(?P<port>:\d+){0,1}(?P<path>/detail/)(?P<id>\d*)`)
+	RegisterAsKnownURL("work-item-details", `(?P<domain>demo.almighty.io)(?P<path>/work-item-list/detail/)(?P<id>\d*)`)
+	RegisterAsKnownURL("localhost-work-item-details", `(?P<domain>localhost)(?P<port>:\d+){0,1}(?P<path>/work-item-list/detail/)(?P<id>\d*)`)
```

Comment:
@baijum I tried running following - 
`./bin/alm-cli show search --q "http://localhost:8088/detail/43" -H localhost:8080 --pp`
that returned me ID 43 object
but 
`./bin/alm-cli show search --q "http://localhost:8088/work-item-list/detail/43" -H localhost:8080 --pp` did return empty.

I know tests passed. but -
```
[pranav@dhcp35-137 almighty-core]$ ./bin/alm-cli show search --q "http://localhost:8088/work-item-list/detail/43" -H localhost:8080 --pp
2016/11/22 14:46:19 [INFO] started id=OMbQEP8m GET=http://localhost:8080/api/search?q=http%3A%2F%2Flocalhost%3A8088%2Fwork-item-list%2Fdetail%2F43
2016/11/22 14:46:19 [INFO] completed id=OMbQEP8m status=200 time=2.351502ms
{
    "data": [],
    "links": {
        "first": "http://localhost:8080/api/search?q=http://localhost:8088/work-item-list/detail/43\u0026page[offset]=0\u0026page[limit]=100",
        "last": "http://localhost:8080/api/search?q=http://localhost:8088/work-item-list/detail/43\u0026page[offset]=0\u0026page[limit]=0"
    },
    "meta": {
        "totalCount": 0
    }
}[pranav@dhcp35-137 almighty-core]$ ./bin/alm-cli show search --q http:///localhost:8088/detail/43" -H localhost:8080 --pp
2016/11/22 14:46:23 [INFO] started id=S86GwzIG GET=http://localhost:8080/api/search?q=http%3A%2F%2Flocalhost%3A8088%2Fdetail%2F43
2016/11/22 14:46:23 [INFO] completed id=S86GwzIG status=200 time=12.432129ms
{
    "data": [
        {
            "fields": {
                "system.assignee": null,
                "system.creator": "d8958274-f741-4558-b117-37cb2ed50d09",
                "system.description": null,
                "system.remote_item_id": null,
                "system.state": "open",
                "system.title": "A catchy Title"
            },
            "id": "43",
            "type": "system.userstory",
            "version": 0
        }
    ],
    "links": {
        "first": "http://localhost:8080/api/search?q=http://localhost:8088/detail/43\u0026page[offset]=0\u0026page[limit]=100",
        "last": "http://localhost:8080/api/search?q=http://localhost:8088/detail/43\u0026page[offset]=0\u0026page[limit]=100"
    },
    "meta": {
        "totalCount": 1
    }
}
```
```